cmd/txncleaner/main.go

```diff
@@ -0,0 +1,47 @@
+// Copyright 2017 Canonical Ltd.
+// Licensed under the LGPLv3, see LICENCE file for details.
+
+package main
+
+import (
+	"flag"
+	"log"
+	"os"
+
+	mgo "gopkg.in/mgo.v2"
+
+	"github.com/juju/txn"
+)
+
+var dbName = flag.String("db", "", "mongo database name (required)")
+var txnsName = flag.String("txns", "txns", "mgo txns collection name")
+var url = flag.String("url", "localhost:27017", "mongo URL")
+
+func main() {
+	flag.Parse()
```

Comment:
Added insecure TLS support similar to [mgopurge](https://github.com/juju/mgopurge/blob/master/src/mgopurge/main.go#L151).
User/pass should already be supported by mgo's URL scheme.
Added a short command description to -help output.